lib/ansible/modules/clustering/kubevirt/kubevirt_template.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kubevirt_template
+
+short_description: Manage KubeVirt templates
+
+description:
+    - Use Openshift Python SDK to manage the state of KubeVirt templates.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    state:
+        description:
+            - Set the to be either I(present) or I(absent).
+        default: "present"
+        choices:
+            - present
+            - absent
+        type: str
+    name:
+        description:
+            - Name of the template.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Namespace where the template exists.
+        required: true
+        type: str
+    objects:
+        description:
+            - List of any valid API objects, such as a I(DeploymentConfig), I(Service), etc. The object
+              will be created exactly as defined here, with any parameter values substituted in prior to creation.
+              The definition of these objects can reference parameters defined earlier.
+        type: list
+    display_name:
+        description:
+            - "A brief, user-friendly name, which can be employed by user interfaces."
+        type: str
+    description:
+        description:
+            - "A description of the template. Include enough detail that the user will understand what is being deployed
+               and any caveats they need to know before deploying. It should also provide links to additional information,
+               such as a README file."
+        type: str
+    long_description:
+        description:
+            - "Additional template description. This may be displayed by the service catalog, for example."
+        type: str
+    provider_display_name:
+        description:
+            - "The name of the person or organization providing the template."
+        type: str
+    documentation_url:
+        description:
+            - "A URL referencing further documentation for the template."
+        type: str
+    support_url:
+        description:
+            - "A URL where support can be obtained for the template."
+        type: str
+    version:
+        description:
+            - "Template structure version."
+        type: str
+    editable:
+        description:
+            - "Extension for hinting at which elements should be considered editable.
+               List of jsonpath selectors. The jsonpath root is the objects: element of the template."
+        type: list
+    default_disk:
```

Comment:
An example of how do you expect the default_* parameters to be used would be nice. Because I have no idea what will those do here.

Reply:
I've added an example.

Reply:
Maybe a short explanation about the result would be nice too. Something like: The default_ parameters define configuration overlays for disks, volumes, ... that will be applied on top of disks, volumes.. during VM creation to define global compatibility and/or performance defaults defined here.

Reply:
Thanks, added.